bundles/org.openhab.binding.systeminfo/src/main/java/org/openhab/binding/systeminfo/internal/handler/SysteminfoHandler.java

```diff
@@ -318,6 +318,13 @@ private State getInfoForChannel(ChannelUID channelUID) {
                 case CHANNEL_SENSORS_FAN_SPEED:
                     state = systeminfo.getSensorsFanSpeed(deviceIndex);
                     break;
+                case CHANNEL_CPU_LOAD:
+                    DecimalType load = systeminfo.getSystemCpuLoad();
+                    if (load != null) {
+                        load = new DecimalType(load.doubleValue() * 100);
```

Comment:
Why not using `PercentType` which is the appropriate state for a percentage state value ?

Reply:
@lolodomo I think this would be the right thing to do.
However, I did the same thing as for the existing usedHeapPercent channel, which was the last channel added and uses Number:Dimensionless with an underlying DecimalType.
But there are also a number of older channels that represent a percentage, with just Number as the type.
Converting them all to PercentType and Number:Dimensionless would be a breaking change. Adding yet another different combination doesn't feel right.

Reply:
@lolodomo What is your view on this?